config/filter.d/softethervpn.conf

```diff
@@ -1,9 +1,16 @@
 # Fail2Ban filter for SoftEtherVPN
 # Detecting unauthorized access to SoftEtherVPN
 # typically logged in /usr/local/vpnserver/security_log/*/sec.log, or in syslog, depending on configuration
+# Config for versions < 0.9 made by quixrick (https://reddit.com/u/quixrick) and jonisc
+# Further reference: http://www.vpnusers.com/viewtopic.php?f=7&t=6375&sid=76707e8a5a16b0c9486a39ba34763901
+
+#Enable multi line support. Doesn't work with versions < 0.9
+[Init]
+maxlines = 2
 
 [INCLUDES]
 before = common.conf
 
 [Definition]
-failregex = ^%(__prefix_line)s(?:(?:\([\d\-]+ [\d:.]+\) )?<SECURITY_LOG>: )?Connection "[^"]+": User authentication failed. The user name that has been provided was "<F-USER>(?:[^"]+|.+)</F-USER>", from <ADDR>\.$
+datepattern = ^%%Y-%%m-%%d %%H:%%M:%%S.%%f
+failregex = IP address: <HOST>.*\n.*User authentication failed.*
```

Comment:
This RE is very weak (no anchors, catch-alls, etc), thus vulnerable.
Also note there is a tag `<SKIPLINES>`, used for skipping multiple lines (especially for `maxlines > 2`).